docs/create/graphql.md

```diff
@@ -20,8 +20,6 @@ type Example @entity {
 }
 ```
 
-**Important: When you make any changes to the schema file please ensure that you regenerate your types directory with the following command `yarn codegen`**
```

Comment:
Is this line removed because we cover it in another place?

Reply:
Yes, it is covered a few lines just above it. line 8 I believe.